doc/themes/scikit-learn-modern/static/css/theme.css

```diff
@@ -71,6 +71,7 @@ a code {
   color: #2878A2;
   border-radius: 0;
   padding: 0;
+  white-space: nowrap;
```

Comment:
Thank you! (I was trying to remember why i had `nowrap` in `code` style in the first place.. it was for the tables!)

Reply:
Now, the problem is that, with these changes, the API elements are not properly rendering for mobile devices:

![2020-06-18 18 01 24](https://user-images.githubusercontent.com/32649176/85044389-d88f5a80-b18d-11ea-93d0-753a2004370b.jpg)

Reply:
And so pages with long tables.

Reply:
Maybe adding:

```css
@media screen and (min-width: 768px) {
  a code {
    white-space: normal;
  }
}
```

is a possible solution?

Reply:
Yea, lets go with the first version for now.

I remember trying to make the table x-scrollable so that it renders nicer on mobile.

Reply:
It seems perfect!

Let us accept the first version and, if you do not mind, I will try to make the tables x-scrollable for mobiles.